### DISCUSSION TITLE
How to set value in input type file using reset().

### DISCUSSION THREAD

**Discussion by dchirag005** [react-hook-form/react-hook-form#8771] [Q&A] [answered]
i was created local environment form API calling.
In API response get image field value only the name of image.

Now how to set image value in file input.

- Here my code 
`<input type="file" className="m-1" style={{paddingTop: '62px'}} 
                                {...register('image', { 
                                    required: true,
                                    onChange: (e) => {
                                        onImageChange(e)
                                    },
                                })}  
                            />`

**Comment by fahadsohail482** [comment, marked as answer]
@dchirag005 you may need an `img` tag to show image. 
To show the image in image tag do the following.
Watch the image using `useWatch` api.
add that watched variable to the `img` 's src attribute.

I hope this might help you.
